sklearn/tests/metadata_routing_common.py

```diff
@@ -146,6 +146,12 @@ def __deepcopy__(self, memo):
     def __copy__(self):
         return self
 
+    def __repr__(self):
+        return "..."
```

Comment:
since repr is now used in error messages, putting `...` here for simplicity. These lists are otherwise pretty ugly and large to print.